src/components/userMenu/Menu.css

```diff
@@ -0,0 +1,11 @@
+@media only screen and (max-width: 640px) {
```

Comment:
use SCSS. Define SCSS variables for screen sizes, colours.

Reply:
Okay.